src/modules/db_redis/doc/db_redis_admin.xml

```diff
@@ -213,6 +213,43 @@ modparam("db_redis", "keys", "version=entry:table_name;location=entry:ruid&amp;u
 				<programlisting format="linespecific">
 ...
 modparam("db_redis", "verbosity", 0)
+...
+				</programlisting>
+			</example>
+		</section>
+
+		<section id="db_redis.p.opt_ssl">
+			<title><varname>opt_ssl</varname> (int)</title>
+			<para>
+				Controls SSL usage while connecting to a remote DB.
```

Comment:
Please do not use "SSL" in any documentation as that protocol is old and deemed insecure. Replace with "TLS".

Setting names in new code should reflect this, regardless of the names used by old libraries.